packages/amplication-cli/src/models.ts

```diff
@@ -749,6 +755,13 @@ export type GitRepository = {
   updatedAt?: Maybe<Scalars['DateTime']>;
 };
 
+export type GitWorkspace = {
```

Comment:
@abrl91 Why all the fields are optional?

Reply:
because it describes a json field